docs-src/cli/README.md

```diff
@@ -0,0 +1,234 @@
+---
+id: readme
+title: temporal readme
+sidebar_label: Install CLI
+description: How to install and configure Temporal CLI.
+tags:
+    - cli
+---
+
+[![Go Reference](https://pkg.go.dev/badge/github.com/temporalio/cli.svg)](https://pkg.go.dev/github.com/temporalio/cli)
+[![ci](https://github.com/temporalio/cli/actions/workflows/ci.yml/badge.svg)](https://github.com/temporalio/cli/actions/workflows/ci.yml)
+
+> ⚠️ Temporal CLI's API is still subject to change. ⚠️
```

Comment:
We need to finalize this ASAP as we're targeting a stable release in a few days.